linux_os/guide/services/sssd/sssd_run_as_sssd_user/rule.yml

```diff
@@ -0,0 +1,29 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Configure SSSD to run as user sssd'
+
+description: |-
+    SSSD processes should be configured to run as user sssd, not root.
+
+rationale: |-
+    To minimize privileges of SSSD processes, they are configured to
+    run as non-root user.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 81062-2
+
+references:
+    ospp: FPT_ACF_EXT.1.1
+    ospp: FDP_ACF_EXT.1.1
```

Comment:
So it seems that the correct reference should be `FMT_SMF_EXT.1`.

Reply:
Fixed in 57abcf27fdf6d46c0251e0fc4a916a5d163a0662.